src/components/avatar-image.tsx

```diff
@@ -1,31 +1,35 @@
 import * as React from "react";
-import Gravatar from "react-gravatar";
 import styled from "styled-components";
 
-const defaultAvatarSizePix = 28;
+const defaultAvatarImgAttributes = {
+  src: "",
+  alt: "аваратрка",
+  width: 28,
+  height: 28,
+};
 
-const StyledGravatar = styled(Gravatar)`
+const StyledImg = styled.img`
   border-radius: 50%;
-  width: ${(props) => props.size}px;
-  height: ${(props) => props.size}px;
   margin-right: 12px;
 `;
 
 interface AvatarImageProps {
-  email?: string | null;
+  src?: string | null;
   size?: number;
+  alt?: string | null;
 }
 
 export const AvatarImage: React.VoidFunctionComponent<AvatarImageProps> = ({
-  email,
+  src,
+  alt,
   size,
```

Comment:
```suggestion
  src = "",
  alt = "аваратрка",
  size = 28,
```

We can get rid of `defaultAvatarImgAttributes` then.

Reply:
That’s OK. Then let’s just change `аваратрка` to `аватарка` 😁

Reply:
BTW using `= default` is a common approach in React apps. Our unit of code is a _component_, which we compose. The closer all component ingredients, the less jumps we need to make when reading or updating code. In your case the default values are still in the same file, so it’s fine.